sql/core/src/test/scala/org/apache/spark/sql/connector/DataSourceV2SQLSuite.scala

```diff
@@ -2004,12 +2005,16 @@ class DataSourceV2SQLSuite
         "USING foo",
         "OPTIONS(",
         "'from' = '0',",
-        "'to' = '1')",
+        "'to' = '1',",
+        "'via' = '2')",
         "PARTITIONED BY (a)",
         "COMMENT 'This is a comment'",
         "LOCATION '/tmp'",
         "TBLPROPERTIES(",
-        "'prop1' = '1')"
+        "'prop1' = '1',",
+        "'prop2' = '2',",
+        "'prop3' = '3',",
+        "'prop4' = '4')"
```

Comment:
The modified test fails with both Scala `2.12` and `2.13` without the change in `ShowCreateTableExec.scala`.